realm-annotations/src/main/java/io/realm/annotations/LinkingObjects.java

```diff
@@ -57,6 +57,7 @@
  *     <li>They are ignored when doing a `copyToRealm().`</li>
  *     <li>They are ignored when doing a `copyFromRealm().`</li>
  *     <li>They are ignored when using the various `createObjectFromJson*` and `createAllFromJson*` methods.</li>
+ *     <li>The listeners on parent object won't be triggered when the linking objects changes.</li>
```

Comment:
Perhaps rephase. I especially find "parent object" confusion since I assume it means the object pointing to this object.

Suggestion: 
Listeners on an object with a @LinkingObject field will not be triggered if the linking objects change, e.g if another object drops a reference to this object.